src/components/MovieCard/MovieCard.tsx

```diff
@@ -3,7 +3,7 @@ import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie,
+  movie: Movie | undefined,
```

Comment:
Which is the case if you don't have the `movie` prop here?
As I see you check `loadedMovie` and render this component if you have a movie, isn't it?

Reply:
There is no reason to render this component if there is no movie.